google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java

```diff
@@ -641,6 +650,15 @@ public static BlobTargetOption metagenerationMatch() {
       return new BlobTargetOption(UnifiedOpts.metagenerationMatchExtractor());
     }
 
+    /**
+     * Returns an option for blob's metageneration match. If this option is used the request will
+     * fail if blob's metageneration does not match the provided value.
+     */
+    @TransportCompatibility({Transport.HTTP, Transport.GRPC})
+    public static BlobTargetOption metaGenerationMatch(long metageneration) {
```

Comment:
metageneration is all one word, so we need to lose the capital `G`.
```suggestion
    public static BlobTargetOption metagenerationMatch(long metageneration) {
```